backend/spec/features/admin/promotion_adjustments_spec.rb

```diff
@@ -8,6 +8,7 @@
       visit spree.admin_path
       click_link "Promotions"
       click_link "New Promotion"
+      expect(page).to have_content("PromotionsNew Promotion")
```

Comment:
This will probably be cleaner if we assert on the title (which uses the same values now). I think drivers other than poltergeist sometimes insert a space between these elements :thinking: 

```
expect(page).to have_title("New Promotion - Promotions")
```

If this doesn't work, though, no big deal.